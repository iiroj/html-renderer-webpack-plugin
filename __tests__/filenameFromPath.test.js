const { filenameFromPath } = require('../index');

const paths = ['/', '/foo', '/bar/'];

it('should generate filenames from paths', () => {
  const filenames = paths.map(path => filenameFromPath(path));
  expect(filenames).toMatchSnapshot();
});
