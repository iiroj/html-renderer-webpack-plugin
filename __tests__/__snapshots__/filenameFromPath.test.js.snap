// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate filenames from paths 1`] = `
Array [
  "index.html",
  "foo.html",
  "bar/index.html",
]
`;
